### PR TITLE
Proofpoint Protection Server README

### PR DESCRIPTION
**Pull request by Itay4**
## Status
- [x] Ready

## Description
 - Added README
 - Fixed the display name
 - Fixed some of the human readable outputs

## Does it break backward compatibility?
   - [x] No

## Must have
- [x] Documentation

### CODE REVIEW

File: Beta_Integrations/ProofpointServerProtection/ProofpointServerProtection.yml [hunk 1]
```diff
@@ -124,7 +124,7 @@ script:
     - contextPath: Proofpoint.SmartSearch.MessageID
       type: string
       description: Search results email message ID
-    - contextPath: Proofpoint.SmartSearch.Proofpoint.SmartSearch.Suborg
+    - contextPath: Proofpoint.SmartSearch.Suborg
```
Comment (yaakovi) [inline Beta_Integrations/ProofpointServerProtection/ProofpointServerProtection.yml]: trigger a build without the BC check using `Utils/trigger_content_build_with_no_backward_check.sh`

**Comment by Itay4** [comment]
@yaakovi what changed in the doc format?

**Comment by yaakovi** [comment]
The is no list of commands anymore.
Also the commands are not numeric ordered

**Comment by Itay4** [comment]
force merging due to bc

**Pull request merged**